src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -35,7 +27,20 @@ case class ClusterRecord(id: Long,
                          destroyedDate: Timestamp,
                          jupyterExtensionUri: Option[String],
                          initBucket: String,
-                         serviceAccountKeyId: Option[String])
+                         machineConfig: MachineConfigRecord,
+                         serviceAccountInfo: ServiceAccountInfoRecord)
```

Comment:
do we need these in the `Record` classes? i can see why we'd need them in the `Cluster` class, since we're json-ifying that, but here too?

Reply:
I added them here because slick doesn't like the `Cluster` case class with >22 members.

Reply:
22 is rapidly becoming my least favorite number.